hydra-node/test/Hydra/BehaviorSpec.hs

```diff
@@ -178,7 +179,10 @@ spec = parallel $ do
               waitUntil [n1, n2] $ HeadIsOpen{headId = testHeadId, utxo = utxoRefs [1, 2]}
 
               send n1 Abort
-              waitUntil [n1] (CommandFailed Abort)
+
+              headState <- queryState n1
+
+              waitUntil [n1] (CommandFailed Abort headState)
```

Comment:
Should: not need this.

We do not need to assert _which_ state we get here, but just that we have _some_ state as information.

A `waitMatch` which only checks the command that failed would be better her IMO (and that we have a state in the response is kind of guaranteed by the types).